change/@uifabric-experiments-2020-08-18-14-59-13-InputWidth.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```

This should be minor since it's adding an API.
```suggestion
  "type": "minor",
```